### ISSUE TITLE
Show static functions in the object function list

### ISSUE THREAD

**Issue by D8H** [4ian/GDevelop#3094]
### Context
* Some actions and conditions that applies to objects are defined statically.

### Actual
* These functions don’t appear on the function list for objects.

### Expected
* These functions should appear on the function list for objects.

### User story
* The user wants to link 2 objects.
* The user selects the 1st object to link.
* The user finds the "Link 2 objects" action in the object action list.

## Solution suggested

The editor could look at the object and behavior parameters to add the static function in the object function list.

## Alternatives considered

Restrict it to only object parameters that are the 1st visible one?

**Comment by 4ian** [comment]
There are a few special cases that are listed in this file: https://github.com/4ian/GDevelop/blob/master/newIDE/app/src/InstructionOrExpression/EnumerateInstructions.js - would it be enough to apply the same thing for the linked objects condition?

**Comment by D8H** [comment]
> There are a few special cases that are listed in this file: https://github.com/4ian/GDevelop/blob/master/newIDE/app/src/InstructionOrExpression/EnumerateInstructions.js - would it be enough to apply the same thing for the linked objects condition?

Ok, It should work for this.
I forgot to explain it in the description, but what about event based functions? Do you think it must be explicitly chosen by the person writing the extension?

**Comment by 4ian** [comment]
Yeah not sure about these. Might be worth adding a flag in the instruction metadata in the future to "consider as object instruction". So we could expose it in extensions directly.
